tools/tox_tools/baseline_calculator/qsar1.xml

```diff
@@ -0,0 +1,46 @@
+<tool id="tt_baseline" name="Baseline toxicity calculator" version="0.1.0+galaxy0">
+    <description>Toxicity prediction tool</description>
+    <requirements>
+        <requirement type="package" version="2.2.1">pandas</requirement>
+    </requirements>
+    <command detect_errors="aggressive"><![CDATA[
+    python '$__tool_directory__/qsar1.py'
+     --function '$function'
+     --csv_input '$csv_input'
+     --functions_csv '$function_csv'
+     --output '$output'
+    ]]></command>
+    <inputs>
+        <param name="function" type="select" label="Process to execute" help="Select pre-set QSAR models or personal linear functions">
```

Comment:
I think a [conditional](https://docs.galaxyproject.org/en/master/dev/schema.html#tool-inputs-conditional) would be nice here. This would allow to show the `function_csv` parameter only if `apply_linear_functions` is selected (it would then also be non-optional).

Reply:
conditional statement was added